OpenSim/Common/ComponentSocket.h

```diff
@@ -896,79 +898,79 @@ class Input : public AbstractInput {
  * not get copied properly if you create a custom copy constructor.
- * We may add support for custom copy constructors with Connectors in the
+ * We may add support for custom copy constructors with Sockets in the
  * future.
  *
- * @see Component::constructConnector()
- * @relates OpenSim::Connector */
-#define OpenSim_DECLARE_CONNECTOR_FD(cname, T, comment)                     \
-    /** @name Connectors                                                 */ \
+ * @see Component::constructSocket()
+ * @relates OpenSim::Socket */
+#define OpenSim_DECLARE_SOCKET_FD(cname, T, comment)                        \
+    /** @name Sockets                                                    */ \
     /** @{                                                               */ \
     /** comment                                                          */ \
-    /** In an XML file, you can set this Connector's connectee name      */ \
-    /** via the <b>\<connector_##cname##_connectee_name\></b> element.   */ \
-    /** See AbstractConnector for more information.                      */ \
-    /** @connectormethods connectConnector_##cname##()                   */ \
+    /** In an XML file, you can set this socket's connectee name         */ \
+    /** via the <b>\<socket_##cname##_connectee_name\></b> element.      */ \
+    /** See AbstractSocket for more information.                         */ \
+    /** @socketmethods connectsocket_##cname##()                         */ \
     OpenSim_DOXYGEN_Q_PROPERTY(T, cname)                                    \
     /** @}                                                               */ \
     /** @cond                                                            */ \
-    PropertyIndex PropertyIndex_connector_##cname##_connectee_name {        \
-        constructConnector_##cname()                                        \
+    PropertyIndex PropertyIndex_socket_##cname##_connectee_name {           \
+        constructSocket_##cname()                                           \
     };                                                                      \
     /* Declare the method used in the in-class member initializer.       */ \
-    /* This method will be defined by OpenSim_DEFINE_CONNECTOR_FD.       */ \
-    PropertyIndex constructConnector_##cname();                             \
+    /* This method will be defined by OpenSim_DEFINE_SOCKET_FD.          */ \
+    PropertyIndex constructSocket_##cname();                                \
     /* Remember the provided type so we can use it in the DEFINE macro.  */ \
-    typedef T _connector_##cname##_type;                                    \
+    typedef T _socket_##cname##_type;                                       \
     /** @endcond                                                         */ \
-    /** @name Connector-related functions                                */ \
+    /** @name socket-related functions                                   */ \
```

Comment:
socket -> Socket ("Connector" was capitalized, presumably because the author wanted a hyperlink in Doxygen). Capitalizing will also match similar comment on L865.